### ISSUE TITLE
exmini csv_logger type in header: 'Entitity_Name'

### ISSUE THREAD

**Issue by ansosafadi** [esmini/esmini#551]
when launching a simulation with the flag --csv_logger, the output csv file contains a type in the header.
"Index [-] , TimeStamp [s] , #1 Entitity_Name [-] , #1 Entitity_ID [-] , #1 Current_Speed [m/s] , #1 Wheel_Angle [deg] , ...."

the header 'Entitity_Name' should actually be "Entity_Name"

**Comment by eknabevcc** [comment]
Yes, true. I see you already contributed a pull request (PR #552) for this fix. Which we have accepted and applied to dev branch. It will then automatically be included in next release.

**Comment by eknabevcc** [comment]
Included in release [v2.37.5](https://github.com/esmini/esmini/releases/tag/v2.37.5)
